content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+Envoy bootstrap configuration.
+  ([Issue #33426](https://github.com/istio/istio/pull/33426)),([Issue #33426](https://github.com/istio/istio/issues/33426))
+
+- **Added** new metric for tracking distribution of configuration resource sizes being pushed by istiod.
+  ([Issue #31772](https://github.com/istio/istio/issues/31772))
+
+- **Added** experimental support for the Kubernetes Multi-Cluster Services (MCS) host (`clusterset.local`).
+This feature is off by default, but can be enabled by setting the following flags in Istio:
+`ENABLE_MCS_HOST` and `ENABLE_MCS_SERVICE_DISCOVERY`. When enabled Istio will include the MCS host as a
+domain in the service's HTTP route. Additionally, Istio will support the MCS host during a DNS lookup.
+For now, the MCS host is just an alias for `cluster.local` and resolves to the same service IP.
+Future work will give the MCS host a separate IP as is defined by the MCS spec.  ([Issue #33949](https://github.com/istio/istio/issues/33949))
+
+- **Added** experimental support for controlling service endpoint discoverability with Kubernetes Multi-Cluster
+Services (MCS). This feature is off by default, but can be enabled by setting the
+`ENABLE_MCS_SERVICE_DISCOVERY` flag in Istio. When enabled, Istio will make service endpoints
+only discoverable from within the same cluster by default. To make the service endpoints within a cluster
+discoverable throughout the mesh, a `ServiceExport` CR must be created within the same cluster as the service
+endpoints. this process can be automated by enabling the Istio flag `ENABLE_MCS_AUTOEXPORT`. With this enabled,
+Istio will automatically create `ServiceExport` in all clusters for each service.
+  ([Issue #29384](https://github.com/istio/istio/issues/29384))
+
+- **Added** Specify `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces for Services, Pods, and Endpoints that istiod processes when pushing xDS updates to improve performance on the data plane.
+  ([Issue #26679](https://github.com/istio/istio/issues/26679))
+
+
+- **Fixed** an issue to enableCoreDump using the sidecar annotation
+ ([reference]( https://istio.io/latest/docs/reference/config/annotations/)) ([Issue #26668](https://github.com/istio/istio/issues/26668))
+
+- **Fixed** an issue where app call itself by podIP in TPROXY mode could neither be intercepted by inbound nor outbound.
+  ([Issue #31095](https://github.com/istio/istio/issues/31095))
+
+- **Fixed** an issue where subject alternate names specified in service entry are not considered while building TLS context.
+  ([Issue #32539](https://github.com/istio/istio/issues/32539))
+
+- **Fixed** a bug where multiple gateways on the same port with `SIMPLE` and `PASSTHROUGH` modes is not working correctly.  ([Issue #33405](https://github.com/istio/istio/issues/33405))
+
+- **Fixed** a bug where Istio config gen fails when sum of endpoint weights over uint32 max.  ([Issue #33536](https://github.com/istio/istio/issues/33536))
```

Comment:
```suggestion
- **Fixed** a bug where Istio config generation fails when the sum of endpoint weights was over uint32 max.  ([Issue #33536](https://github.com/istio/istio/issues/33536))
```

@istio/wg-docs-maintainers Should we define what uint32 max is or is this assumed knowledge?